limbo/config.py

```diff
@@ -0,0 +1,50 @@
+import os
+from backports import configparser as confparser
```

Comment:
Where does backports come from?

Reply:
Gives some backports of features from newer versions of Python to previews: https://pypi.python.org/pypi/configparser